src/main/java/org/jenkinsci/plugins/workflow/log/TaskListenerDecorator.java

```diff
@@ -219,6 +219,17 @@ private static class ConsoleLogFilterAdapter extends TaskListenerDecorator {
 
     }
 
+    private static OutputStream decorateAll(OutputStream base, List<TaskListenerDecorator> decorators) {
+        for (TaskListenerDecorator decorator : decorators) {
+            try {
+                base = decorator.decorate(base);
+            } catch (Throwable x) {
+                LOGGER.log(Level.WARNING, null, x);
```

Comment:
Whilst this defends against the issue I am concerned this error will never be picked up - most people are not looking at jenkins logs.  Without a build to log at least prompt the build user to ask the admin to look at the logs.

Having said that - I have no idea how we would get the charset to print something here - without the charset we risk corrupting the entire output :(

Reply:
⇒ d191982eb38dd0dc85a75c2378e17b3a278e1ad8

> the charset to print something here

⇒ 696e4f8900771738d075bbc38524ff0de7899a77